src/lazy/text/matched.rs

```diff
@@ -1232,7 +1232,7 @@ mod tests {
             let expected: Int = expected.into();
             let encoding_context = EncodingContext::empty();
             let context = encoding_context.get_ref();
-            let mut buffer = TextBuffer::new(context, data.as_bytes(), true);
+            let mut buffer = TextBuffer::new(context, data.as_bytes());
```

Comment:
🪧 `TextBuffer::new` is only used in the unit tests where `is_final_data` is always `true`, so I removed that argument.